misc/io.github.grangerhub.Tremulous.desktop

```
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Tremulous
+Comment=Aliens vs Humans, First Person Shooter game with elements of Real Time Strategy (GrangerHub version)
+Icon=io.github.grangerhub.Tremulous.png
```

```suggestion
Icon=io.github.grangerhub.Tremulous
```

The desktop icon should not contain a file type suffix.